src/components/RateArticles/RateArtcles.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import StarRatingComponent from 'react-star-rating-component';
+import PropTypes from 'prop-types';
+
+class ArticleRating extends React.Component {
+  state = {};
+
+  onStarClick = nextValue => {
+    const { article, starClick } = this.props;
+    const slug = article.slug;
+    starClick(slug, { rate: nextValue }, rating =>
+      this.setState({
+        rating
```

Comment:
Hello @Philipotieno ,
You are not using the state element `rating` so you could do away with it. On matters that, the method:
```
  onStarClick = nextValue => {
    const { article, starClick } = this.props;
    const slug = article.slug;
    starClick(slug, { rate: nextValue }, rating =>
      this.setState({
        rating
      })
    );
  };
```
In `RateArticles.jsx` is not actually being used.